### PR TITLE
Release 0.16.0

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 addSbtPlugin(
-  "com.47deg" % "sbt-org-policies" % "0.7.4" exclude ("io.get-coursier", "sbt-coursier"))
+  "com.47deg" % "sbt-org-policies" % "0.7.5" exclude ("io.get-coursier", "sbt-coursier"))
```
Comment (BenFradet) [inline project/plugins.sbt]: @juanpedromoreno is this all that needed or do I still need to fiddle with .travis.yml

Comment (juanpedromoreno) [reply project/plugins.sbt]: Mmm, I'm not fully sure. @suhasgaddam Does 0.7.5 version fix the realese issue? Thanks.

Comment (suhasgaddam) [reply project/plugins.sbt]: Yes, `0.7.5` fixes `orgAfterCISuccess` for all normal builds.

**Comment by BenFradet** [comment]
Nice! Thanks @suhasgaddam @juanpedromoreno .

**Comment by BenFradet** [comment]
mmh apparently it didn't work, any input on how I should proceed to get 0.16.0 out? Should I follow https://github.com/47deg/sbt-org-policies/blob/2c89b98d9366b626a978e3ff17cbc25e561ea48b/.travis.yml#L31-L44 ?

**Pull request merged**